src/WpfMath/TexFormulaParser.cs

```diff
@@ -753,7 +752,7 @@ private Color ReadColorModelData(SourceSpan value, ref int position)
             var primesRowSource = new SourceSpan(
                 value.SourceName,
                 value.Source,
-                primesRowAtom.Source.Start,
+                primesRowAtom.Source!.Start, // Nullable TODO: This might need null checking
```

Comment:
This is alright; `primesRowAtom` always has a source here, because we create it with a non-nullable source.

Reply:
Ah, missed the ctor above.